modules/apps/frontend-theme/frontend-theme-classic/src/css/portlet/_portlet_decorators.scss

```diff
@@ -4,10 +4,7 @@
 	}
 
 	.portlet-content {
-		background-color: $portlet-decorate-bg;
-		border: $portlet-decorate-border;
 		border-radius: 0.25rem;
-		padding: 24px;
```

Comment:
Replaced the manual overwrite with variables override, we have the variables ready https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-theme/frontend-theme-styled/src/main/resources/META-INF/resources/_styled/css/portlet/_variables.scss